web/templates/symptom_tracker/new.html.eex

```diff
@@ -3,7 +3,7 @@
 
   <%= form_for @changeset, symptom_tracker_path(@conn, :create), fn f -> %>
     <div class="">
-      <%= label f, "Rate how much this problem or difficulty affected you today:", class: "f5 b db mb2" %>
+      <%= label f, "Rate how much this problem or difficulty effected you today:", class: "f5 b db mb2" %>
```

Comment:
I'm happy to _merge_ it because _presumably_ that request would not have been made without some _certainty_ ... then we can let the (_eagle-eyed_) "_stakeholders_" feedback. 😉